selection_sort/cpp/Cpp selection_sort.cpp

```diff
@@ -0,0 +1,35 @@
+/* C++ Program - Selection Sort */
+		
+#include<iostream.h>
+#include<conio.h>
+void main()
+{
+	clrscr();
+	int size, arr[50], i, j, temp;
+	cout<<"Enter Array Size : ";
+	cin>>size;
+	cout<<"Enter Array Elements : ";
+	for(i=0; i<size; i++)
+	{
+		cin>>arr[i];
+	}
+	cout<<"Sorting array using selection sort...\n";
+	for(i=0; i<size; i++)
+	{
+		for(j=i+1; j<size; j++)
+		{
+			if(arr[i]>arr[j])
+			{
+				temp=arr[i];
+				arr[i]=arr[j];
+				arr[j]=temp;
+			}
+		}
+	}
+	cout<<"Now the Array after sorting is :\n";
+	for(i=0; i<size; i++)
+	{
+		cout<<arr[i]<<" ";
+	}
+	getch();
+}
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmphixsekeh/selection_sort/cpp/Cpp selection_sort.cpp
+++ b/tmp/tmphixsekeh/selection_sort/cpp/Cpp selection_sort.cpp
@@ -32,4 +32,4 @@
 		cout<<arr[i]<<" ";
 	}
 	getch();
-}+}
```